templates/email/highwaysengland/submit.html

```diff
@@ -40,6 +40,10 @@ <h2 style="[% h2_style %]">[% report.title | html %]</h2>
       <strong>Section:</strong>
         [% report.get_extra_field_value('sect_label') %]
     </p>
+    <p style="[% secondary_p_style %]">
+      <strong>Heard from:</strong>
+        [% report.get_extra_field_value('where_hear') %]
```

Comment:
report_form_extras puts in metadata, not extra fields. Of course, what flows from that is - this needs a test :)